build.gradle

```diff
@@ -105,54 +104,16 @@ task deobfJar(type: Jar, dependsOn: classes) {
         ]
     }
 }
-/* Jar for just the API - Stays deobfuscated so it can be added as a mod project dependency */
-task modJar(type: Jar, dependsOn: classes) {
-    mustRunAfter = ['jar']
-    from sourceSets.main.output
-    classifier = 'examplemod'
-    include 'example/**'
-    include 'assets/**'
-    manifest {
-        attributes = [
-                "Manifest-Version"      : "1.0",
-                "Implementation-Title"  : project.title_example,
-                "Implementation-Version": project.version,
-                "Implementation-URL"    : project.git_website
-
-        ]
-    }
-}
 
 remapJar {
-    classifier = 'fabric'
+    archiveClassifier = 'fabric'
```

Comment:
Let's remove this classifier. 
So it is `journeymap-api-1.18.2-1.8-fabric-SNAPSHOT.jar` not `journeymap-api-1.18.2-1.8-fabric-SNAPSHOT-fabric.jar`